### PR TITLE
Fix Backtesting  Analysis Column Wrong

### PR DESCRIPTION
**Pull request by Undertoned**
## Summary

I find wrong column when i analyze my trade data , so just fix it

Solve the issue: #___

## Quick changelog

- <change log 1>
- <change log 1>

### CODE REVIEW

**Comment by froggleston** [comment]
I'm not sure what was "wrong" here? I understand adding the function argument typing, but what is the purpose of rearranging the column order?

File: freqtrade/data/entryexitanalysis.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def _do_group_table_output(bigdf, glist):
 
         else:
             agg_mask = {'profit_abs': ['count', 'sum', 'median', 'mean'],
-                        'profit_ratio': ['sum', 'median', 'mean']}
+                        'profit_ratio': ['median', 'mean', 'sum']}
             agg_cols = ['num_buys', 'profit_abs_sum', 'profit_abs_median',
                         'profit_abs_mean', 'median_profit_pct', 'mean_profit_pct',
                         'total_profit_pct']
```
Comment (xmatthias) [inline freqtrade/data/entryexitanalysis.py:126]: @froggleston 
i was doubting the same at first - but:
if you compare the changed fields (`agg_mask`) with `agg_cols` - you'll notice that the sequence is wrong.
(total_profit_pct != profit_ratio.mean) ...

Comment (froggleston) [reply freqtrade/data/entryexitanalysis.py:126]: Oh great spot @Undertoned ! LGTM.

**Review by xmatthias — APPROVED** [review]
great spot! :+1:

**Pull request merged**